### PR TITLE
docs: update active_versions.json

### PR DESCRIPTION
**Pull request by sam-heilbron**
# Description

Ensure that docs point to the latest stable version

# Context
1.18 Is now GA

## Interesting decisions
-

## Testing steps
-[x] Docs action runs successfully, and 1.18 version is included
-
## Notes for reviewers
-

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/active_versions.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
 {
-  "latest": "v1.17.x",
+  "latest": "v1.18.x",
   "versions": [
     "main",
-    "v1.17.x"
+    "v1.18.x"
   ],
   "oldVersions": [
+    "v1.18.x",
```
Comment (jenshu) [inline docs/active_versions.json:8]: why is 1.18 also under oldVersions?

**Comment by Nadine2016** [comment]
@sam-heilbron I have this change in my PR https://github.com/solo-io/gloo/pull/10475

**Comment by sam-heilbron** [comment]
> @sam-heilbron I have this change in my PR #10475

Oh great! I will close mine

**Pull request closed without merging**